Dockerfile

```diff
@@ -30,6 +30,7 @@ RUN --mount=from=binary,target=/build \
   --mount=type=bind,target=/src \
   mkdir -p /out \
   && cp /build/wait4x /src/README.md /src/LICENSE . \
+  && if [$TARGETOS = "windows/amd64" ] || [$TARGETOS = "windows/arm64" ] then mv wait4x wait4x.exe fi \
```

Comment:
The value of "TARGETOS" is "windows", not "windows/amd64" or "windows/arm64".

But I think It's better to move the this logic to the Makefile (after [line 87](https://github.com/atkrad/wait4x/blob/main/Makefile#L87)) and check only this condition: `$GOOS == "windows"`

Reply:
Thanks for the feedback, will adjus the Makefile accordingly

Reply:
The makefile should reflect the needed changes. However I'm not sure if `$GOOS` is available there